posts/getting-started-with-type-hinting-in-python.md

```diff
@@ -63,6 +63,7 @@ Running Mypy on the code will return the following errors.
 
 <Highlighter
 language='bash'
+output='true'
```

Comment:
I am not totally sold on this being the best way to do this. Output blocks should not have code copy functionality, but we could just render them as regular code blocks, without the `Highlighter`.

Reply:
@ltagliaferri what do you think — which way would you prefer? I guess I may have a slight preference for option 2, but happy to go with whatever you want. Either way, I will need to update output blocks on existing tutorials as part of this PR.

Reply:
option 2 sounds good to me

Reply:
Went with a different approach (updated the comment above with more details), since I realized that we do ultimately want highlighting on output blocks too, and I think it makes sense to bundle that Jira into this work.